server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlService.java

```diff
@@ -31,21 +36,24 @@ public class AdmissionControlService {
     public final AdmissionControlSettings admissionControlSettings;
     private final ConcurrentMap<String, AdmissionController> ADMISSION_CONTROLLERS;
     private static final Logger logger = LogManager.getLogger(AdmissionControlService.class);
-    private final ClusterSettings clusterSettings;
+    private final ClusterService clusterService;
     private final Settings settings;
 
+    private ResourceUsageCollectorService resourceUsageCollectorService;
+
     /**
      *
      * @param settings Immutable settings instance
-     * @param clusterSettings ClusterSettings Instance
+     * @param clusterService ClusterSettings Instance
```

Comment:
Same comment everywhere - lets update comments to reflect the function parameters